docs/meshcloud.project-metering.md

```diff
@@ -30,11 +30,16 @@ Additionally an amount can be set on the payment method, which indicates how muc
 
 A tenant usage report provides usage information of one tenant (project representation in a specific platform). E.g. all resources like VMs, Storage, Public IPs, etc are shown in this report.
 
-Tenant Usage Reports can be accessed in the tenant control plane under **Finacials** > **Usage Reports**. For every period (e.g. a month),
-a tenant usage report is created. You also have access to the current period (e.g. the current month), to always have full control of
+Tenant Usage Reports can be accessed in the tenant control plane under **Finacials** > **Usage Reports**.
+meshStack generates a tenant usage report for every usage period (monthly).
+
+> meshStack defines **usage period** as starting from the first day of a month to the last day of the month in UTC.
+
+You also have access to a preview report of the current period (e.g. the current month), to always have full control of
 your resources and their usage.
 
-From a Tenant Usage Report you can also access a Detailed Report to see every single usage the resources created. That means you can e.g. see whether a resource was running for 3 hours, stopped for 5 hours and then started again for another 6 hours.
+From a Tenant Usage Report you can also access a Detailed Report to see every single usage the resources created.
```

Comment:
From a Tenant Usage Report you can also access a Detailed Report to see every single usage the resources have created.